app/controllers/dashboard/protocols_controller.rb

```diff
@@ -90,6 +89,7 @@ def show
   end
 
   def new
+    @action = params['action']
```

Comment:
You can access the controller action using #action_name (http://stackoverflow.com/questions/15841617/if-action-name-rails)